src/class-wp-importer-cron.php

```diff
@@ -11,6 +11,10 @@
 
 defined( 'ABSPATH' ) || exit;
 
+if ( class_exists( 'WP_Importer_Cron' ) ) :
```

Comment:
A check for this was removed when moving things to the `src/` directory.

WordPress.com implements it's own `WP_Importer_Cron`.

Reply:
is there a reason for not using `{}`?